app/decorators/classification_decorator.rb

```diff
@@ -1,5 +1,5 @@
 class ClassificationDecorator < MiqDecorator
   def fonticon
-    'pficon pficon-folder-close'
+    category? ? 'pficon pficon-folder-close' : 'fa fa-tag'
```

Comment:
You'll have to rebase here... when doing that, please keep in mind that the `category?` check can only be in `fonticon`, not `self.fonticon`.